spec/mongo/index/view_spec.rb

```diff
@@ -196,7 +196,70 @@
         expect(view.get('random_name')).to_not be_nil
       end
     end
-  end
+
+    context 'when providing an invalid partial index filter' do
+
+      let(:spec) do
+        { random:1 }
```

Comment:
We would put in a space in between the key and the value in a hash. For example:

```ruby
{ random: 1}
```

and here's an existing example:
https://github.com/aherlihy/mongo-ruby-driver/blob/1034-partial-index/spec/mongo/index/view_spec.rb#L107